app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -215,10 +223,12 @@ export function Filters(props: FiltersProps) {
       }, // Include an empty object as payload
     });
   };
-
   if (filterData) {
     return (
       <div className={props.className}>
+        <div className="sticky p-2 z-10 top-0 w-full bg-white">
```

Comment:
bg-white to not have a gross looking half overlap with the elements below the fixed filter bar.

Reply:
You do not need to make this sticky, you can just take this out of the scrollable filters part, and then have that be on top and then make the internal div with the other filters overflow-scroll.